src/AI/JakeMan.java

```diff
@@ -52,6 +52,7 @@ public AIMaker getAIInfo()
   private static final double FIRSTSTRIKE_ON_THREAT_WEIGHT = 2.0;
   private static final int    STAY_UNHURT_BIAS = 1000;
   private static final int    STAY_ALIVE_BIAS = 2000;
+  private static final int    BIG_THREAT_THRESHOLD = 80; // Enemy health at which I double the expected value of dealing damage
```

Comment:
This is to make attacking stronger/high-HP defenders more likely? As in, it's twice as valuable to chip down threats to below 9 HP as it is to do damage below that? Sure, I guess.

Reply:
It's basically an incentive to firststrike the most threatening thing, since 2 5 HP tanks is usually less valuable immediately than 1 full HP tanks (because of how counters/etc work)